pydatastructs/linear_data_structures/algorithms.py

```diff
@@ -1369,3 +1372,197 @@ def bubble_sort(array, **kwargs):
         array._modify(force=True)
 
     return array
+
+def linear_search(array, elem, **kwargs):
+    """
+    Implements linear search algorithm.
+
+    Parameters
+    ==========
+
+    array: OneDimensionalArray
+        The array which is to be sorted.
+    elem: same as that of the array
+    start: int
+        The starting index of the portion
+        which is to be searched.
+        Optional, by default 0
+    end: int
+        The ending index of the portion which
+        is to be searched.
+        Optional, by default the index
+        of the last position filled.
+    backend: pydatastructs.Backend
+        The backend to be used.
+        Optional, by default, the best available
+        backend is used.
+
+    Returns
+    =======
+
+    output: int
+        The index of elem if found.
+        If not found, returns -1.
+
+    Examples
+    ========
+
+    >>> from pydatastructs import OneDimensionalArray, linear_search
+    >>> arr = OneDimensionalArray(int,[3, 2, 1])
+    >>> out = linear_search(arr, 2)
+    >>> print(out)
+    1
```

Comment:
```suggestion
    >>> linear_search(arr, 2)
    1
```
Above should work. Can you try?

Reply:
Yes Sure

Reply:
I did not test it in conda environment so i am unfamiliar. But it works as intended, I checked in vs code.

Reply:
I think only Python >= 3.5 is needed for testing. Doesn't matter whether it comes from conda, or VS Code.

Reply:
See the testing section of README.

Reply:
Yes, I havent done it like that, I just ran sample codes to check. But it works

Reply:
Ok, will do